Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -563,16 +563,19 @@ HalfRegex: !simpleRegex
   def: (((een)\s*)|\b)(?<half>(half|halve)\s+(?<unit>jaar|maand|week|dag|uur|halfuur)|(?<unit>halfuur))\b
 ConjunctionRegex: !simpleRegex
   def: \b((en(\s+voor)?)|plus)\b
-HolidayRegex1: !nestedRegex
-  def: \b(?<holiday>(goede\s+vrijdag|pasen|kerst|kerstavond|kerstmis|thanksgiving|halloween|nieuwjaar|bevrijdingsdag))(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
+HolidayList1: !nestedRegex
+  def: (?<holiday>goede\s+vrijdag|pasen|kerst|kerstavond|kerstmis|thanksgiving|halloween|nieuwjaar|bevrijdingsdag|pinksteren|oude?jaarsavond)
   references: [ YearRegex, RelativeRegex ]
-# TODO: re-add oude?jaardsavond
-HolidayRegex2: !nestedRegex
-  def: \b(?<holiday>(nationale dodenherdenking|nationale herdenking|dodenherdenking|dag van de leraar|dag van de arbeid|martin luther kingdag|mlkdag))(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
+HolidayList2: !nestedRegex
+  def: (?<holiday>nationale dodenherdenking|nationale herdenking|dodenherdenking|dag van de leraar|dag van de arbeid|dag|yuandan|valentijn)
   references: [ YearRegex, RelativeRegex ]
-HolidayRegex3: !nestedRegex # -dag suffix
-  def: \b(?<holiday>(yuandan|valentijnsdag|valentijn|oude?jaarsavond|nieuwjaarsdag|eerste paasdag|tweede paasdag|prinsjesdag|koningsdag|koninginnedag|bevrijdingsdag|hemelvaartsdag|eerste kerstdag|1e kerstdag|tweede kerstdag|2e kerstdag|vaderdag|moederdag|meisjesdag|amerikaanse onafhankelijkheidsdag|onafhankelijkheidsdag|nederlandse veteranendag|veteranendag|boomplantdag|boomfeestdag))(\s+(van\s+|in\s+)?({YearRegex}|{RelativeRegex}\s+jaar))?\b
+HolidayList3: !nestedRegex # -dag suffix
+  def: (?<holiday>(martin luther king|mlk|valentijns|nieuwjaars|(eerste|1e|tweede|2e)\s+paas|prinsjes|konings|koninginne|bevrijdings|hemelvaarts|(eerste|1e|tweede|2e)\s+kerst|vader|moeder|meisjes|(amerikaanse\s+)?onafhankelijkheids|onafhankelijkheids|nederlandse veteranen|veteranen|boomplant|boomfeest)dag)
```

Comment:
This revised regex currently has the issue that it matches `dag` as well. 

```
{
  "Text": "dag",
  "Start": 0,
  "End": 2,
  "TypeName": "datetimeV2.date",
  "Resolution": null
}
```

Reply:
It looks like "dag" is being matched elsewhere. Not in the H3 regex.

Reply:
Would be my thought as well, however this behaviour has been introduced since this PR. I will have a better look, otherwise I will revert the change for HolidayList3.